maven-plugins/run-automation-package-sample/src/test/java/step/examples/plugins/StepRunnerTestKeywords.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (C) 2020, exense GmbH
+ *
+ * This file is part of STEP
+ *
+ * STEP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * STEP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with STEP.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package step.examples.plugins;
+
+import step.handlers.javahandler.AbstractKeyword;
+import step.handlers.javahandler.Keyword;
+
+public class StepRunnerTestKeywords extends AbstractKeyword {
+
+	@Keyword
+	public void callExisting() {
+		System.out.println("My first keyword is called!");
```

Comment:
avoid system out in the code sample, a output.add would do .

Reply:
@david-stephan What do you think about using a realistic sample based on open cart here?

Reply:
@jeromecomte @iegorov777 yes I think it's always better / easier to understand with concrete examples and would also allow to reuse it in our doc and tutorials. I think we could use a simple "benchmark" scenario" (might make sense in the context of CI/CD). For instance following plan (the keyword can be copied over form the playwright tutorial)

// ThreadGroup Buy MacBook in OpenCart
ThreadGroup Threads=1 Pacing=0 MaxDuration=0 Iterations=10
    "Buy MacBook in OpenCart"
End
PerformanceAssert Measurement="Buy MacBook in OpenCart" Value=10000